### PR TITLE
feat: isUndefined

### PR DESCRIPTION
**Pull request by jsertx**
### Description:
add isUndefined

### CODE REVIEW

**Review by btrdnch — APPROVED** [review]
looks good
please resolve conflicts first

**Review by btrdnch — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -75,6 +75,8 @@ module.exports = {
   isObjectLike,
   isPlainObject,
   isString,
+  isUndefined,
+  flow,
```
Comment (vigan-abd) [inline index.js]: flow is befere isUndefined alphabetically

File: test/isUndefined.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+describe('isUndefined', () => {
+  it('should return true for undefined', () => {
+    assert.ok(isUndefined(undefined))
+  })
+
+  it('should return true for no param', () => {
+    assert.ok(isUndefined())
+  })
+})
```
Comment (vigan-abd) [inline test/isUndefined.js]: not sure if this is full coverage as I haven't run tests locally but we should add also case when it returns false

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,5 @@
+#1.18.0
+- feat: isUndefined
```
Comment (vigan-abd) [inline CHANGELOG:2]:
```suggestion
#1.18.0
- feat: isUndefined

```

**Review by vigan-abd — CHANGES_REQUESTED** [review]

File: test/isUndefined.js (hunk 2 above)
Comment (jsertx) [reply test/isUndefined.js]: you are right, I have added full coverage for falsy values

**Review by vigan-abd — APPROVED** [review]

**Pull request merged**